specs/ChainSpec/Consensus.md

```diff
@@ -167,11 +167,11 @@ A block $B$ is final in $\operatorname{chain}(T)$, where $T \ge B$, when either
 
 ## Epoch switches
 
-There's a parameter $epoch_length \ge 3$ that defines the minimum length of an epoch. Suppose that a particular epoch $e_cur$ started at height $h$, and say the next epoch will be $e_next$. Say $\operatorname{BP}(e)$ is a set of block producers in epoch $e$. Say $\operatorname{last_final}(T)$ is the highest final block in $\operatorname{chain}(T)$. The following are the rules of what blocks contain approvals from what block producers, and belong to what epoch.
+There's a parameter $\operatorname{epoch\textunderscore length} \ge 3$ that defines the minimum length of an epoch. Suppose that a particular epoch $e_{cur}$ started at height $h$, and say the next epoch will be $e_{next}$. Say $\operatorname{BP}(e)$ is a set of block producers in epoch $e$. Say $\operatorname{last\textunderscore final}(T)$ is the highest final block in $\operatorname{chain}(T)$. The following are the rules of what blocks contain approvals from what block producers, and belong to what epoch.
```

Comment:
`\textunderscore` renders literally

<img width="983" alt="image" src="https://user-images.githubusercontent.com/304265/192308359-dec51c49-bd33-41ab-b1ba-081104da6f12.png">

Is there another way to fix it? 🤔